Emergency fixes

tcgj opened the pull request:
Latest commit: Someone please help me test the monster movement and attack. I tried to test it myself but my computer crashed maybe 5 times. If it doesn't work...either let me know or just reset by 1 commit. Keep the other two.

- Fix tower capturing not completing on clients due to interpolation, and added a timeout for hosting button.

tcgj committed:
Fix tower capture not being completed for clients

tcgj committed:
Add host button timeout

tcgj committed:
Fix monsters not synchronising movement and spawning multiple damage

heze8 committed:
Fixed plant death anim

heze8 reviewed (approved):
I added some fixes as well and tested it locally. Nothing gamebreaking I believe.

heze8 committed:
Make monster retreat when player is too far away

heze8 committed:
Update semicolon

tcgj committed:
Kill all monster instances if it dies in any client

heze8 committed:
Change trigger target to nearest player

heze8 committed:
Fix float

tcgj committed:
Fix skill handler picking up network player skills

heze8 committed:
Fix not moving

tcgj committed:
Fix ui skill handler not removing correct skill

tcgj committed:
Fix monster attack animation not triggering

fatclarence committed:
increase capsule collider on monster

fatclarence committed:
increase capsule collider on all monsters again

tcgj committed:
Fix network player attempting to access player movement

heze8 committed:
Made the player detection update onTriggerstay as well

heze8 committed:
Check for reference of nearest player in couroutine update

fatclarence committed:
increase height of mushroom capsule collider

heze8 committed:
Add last monster to scene

fatclarence committed:
edit capsule colliders of in-scene monsters

tcgj committed:
Minor game fixes

tcgj committed:
Add character controller null check

fatclarence committed:
fix mushroom colliders to improve damage detection

fatclarence committed:
fix shuriken colliding with player and move mushroom monster

tcgj committed:
Set player without display name as Guest

heze8 committed:
Check for null

tcgj committed:
Make tower fill gauge relative to fraction of players in zone

tcgj committed:
Remove buggy hairstyle from customisation

Eclmist committed:
Playtest Fixes e6.2.1

- Fixed crack in trees
- Fixed shadows newr towers
- Fixed tower glow not double sided
- Fixed combat blocking
- Fixed notifications not destroying after closing
- Fixed notifications not clearly visible
- Fixed tower notifications playing even after capture
- Fixed blurry UI text
- Fixed monster collider not disabling immediately after death
- Fixed wind

- Add name tags to players
- Add more monsters to main scene

Eclmist committed:
Removed junk from master

tcgj committed:
Change tower capture message

Eclmist committed:
Fix terrain hole texture sampling
